services/tenant-ui/frontend/src/components/reservation/Status.vue

```diff
@@ -57,14 +57,16 @@
       }}</small>
     </div>
 
-    <Button type="submit" class="w-full my-2" label="Check Status" />
+    <Button
+      type="submit"
+      class="w-full my-2 check-status-button"
+      label="Check Status"
+      :icon="spinnerComputed()"
```

Comment:
See comment below

Reply:
Don't know how I missed that. Is it not in the primevue docs?